content/en/agent/guide/autodiscovery-with-jmx.md

```diff
@@ -71,16 +71,17 @@ The Datadog-Kafka integration example below leverages JMX to collect metrics and
 
 1. Get the configuration files `conf.yaml` and `metrics.yaml` associated to the Datadog-Kafka integration. They can be found in your [integrations configuration directory][1]. Find below the list of JMX based integration with their associated files:
 
-    | Integration         | Metrics file       | Configuration file      |
-    |---------------------|--------------------|-------------------------|
-    | [ActiveMq][2]       | [metrics.yaml][3]  | [conf.yaml.example][4]  |
-    | [Cassandra][5]      | [metrics.yaml][6]  | [conf.yaml.example][7]  |
-    | [Hive][8]           | [metrics.yaml][9]  | [conf.yaml.example][10] |
-    | [Jboss Wildfly][11] | [metrics.yaml][12] | [conf.yaml.example][13] |
-    | [Kafka][14]         | [metrics.yaml][15] | [conf.yaml.example][16] |
-    | [Solr][17]          | [metrics.yaml][18] | [conf.yaml.example][19] |
-    | [Presto][20]        | [metrics.yaml][21] | [conf.yaml.example][22] |
-    | [Tomcat][23]        | [metrics.yaml][24] | [conf.yaml.example][25] |
+    | Integration             | Metrics file       | Configuration file      |
+    |-------------------------|--------------------|-------------------------|
+    | [ActiveMq][2]           | [metrics.yaml][3]  | [conf.yaml.example][4]  |
+    | [Cassandra][5]          | [metrics.yaml][6]  | [conf.yaml.example][7]  |
+    | [Confluent Platform][5] | [metrics.yaml][29] | [conf.yaml.example][30]  |
```

Comment:
The link for confuent platform is the same as cassadra, is this expected ?

Reply:
Good catch. Thx. Fixed.